packages/plugin-network-breadcrumbs/test/network-breadcrumbs.test.ts

```diff
@@ -501,7 +514,7 @@ describe('plugin: network breadcrumbs', () => {
 
     const request = new XMLHttpRequest()
     request.open('GET', '/')
-    request.send(false, 200)
+    request.send(undefined, false, 200)
```

Comment:
why do we need to pass undefined here?

Reply:
That's the request body - the mock `XMLHttpRequest` class had some existing optional parameters to set the desired failure state and status code but the real `XMLHTTPRequest.send` method just accepts a single (optional) `body` parameter - without this the plugin would interpret the request body as `false` and try to calculate the content length for that.